tests/tt_eager/python_api_testing/sweep_tests/pytests/tt_dnn/test_copy.py

```diff
@@ -140,3 +140,29 @@ def test_run_clone_op(
             device,
             test_args,
         )
+
+    def test_run_typecast_op(
```

Comment:
Should add tests for actually testing different input/output dtypes. (I also missed adding this for clone)